app/models/metric/statistic.rb

```diff
@@ -1,15 +1,18 @@
 module Metric::Statistic
   def self.calculate_stat_columns(obj, timestamp)
-    return {} unless obj.respond_to?(:container_groups)
-    return {} unless obj.respond_to?(:container_images)
-
     capture_interval = Metric::Helper.get_time_interval(obj, timestamp)
-    container_groups = ContainerGroup.where(:ems_id => obj.id).or(ContainerGroup.where(:old_ems_id => obj.id))
+    stats = {}
+
+    if obj.respond_to?(:all_container_groups)
```

Comment:
looks like this fixes a bug (previously stat_container_group_delete_rate for ems were always 0)